prow/plugins/trigger/push.go

```diff
@@ -17,12 +17,31 @@ limitations under the License.
 package trigger
 
 import (
+	"fmt"
+
 	"k8s.io/test-infra/prow/github"
 	"k8s.io/test-infra/prow/kube"
 	"k8s.io/test-infra/prow/plank"
 )
 
+func notifyOnSlackIfManualMerge(sc slackClient, pe github.PushEvent) error {
+	if sc == nil {
+		return nil
+	}
+	if pe.Pusher.Name != "k8s-merge-robot" && pe.Sender.Login != "k8s-merge-robot" {
+		message := fmt.Sprintf("Warning: %s manually merged %s", pe.Pusher.Name, pe.Compare)
+		if err := sc.WriteMessage(message, "sig-contribex"); err != nil {
+			return err
+		}
+		return sc.WriteMessage(message, "kubernetes-dev")
+	}
+	return nil
+}
+
 func handlePE(c client, pe github.PushEvent) error {
+	if err := notifyOnSlackIfManualMerge(c.SlackClient, pe); err != nil {
```

Comment:
Please move this to its own plugin. The trigger plugin is for triggering test jobs.